src/test/resources/no/nav/familie/ef/sak/beregning_barnetilsyn_mangeperioder_maxbeløp_barn.feature

```diff
@@ -0,0 +1,59 @@
+# language: no
+# encoding: UTF-8
+
+# Gitt Perioder 2021:
+# JAN --------------------- JULI --------------------------- DES (Utgiftsperioder = 2)
+
+# Gitt Perioder 2022:
+# JAN-----------------------AUG-------------------------- DES (Utgiftsperioder )
+#         MAI--------------------------SEP                    (Kontantstøtte)
+#                   JUL--------------------------NOV          (Tillesstønad)
+#
+# JAN-----MAI-------JUL-----AUG--------SEP-------NOV----- DES (Resultat = 6)
+
+# Formel beløp når vi treffer maxgrense er gitt antall barn og max-sats for perioden
+# Eksempel overgang fra 1 til 2 barn i 2021 4195->5474
+# Eksempel på satsendring fra 2021 til 2022:  5474 -> 5545 (med samme antall barn)
+
+# Til diskusjon: hull i perioder 07-09
+
+# Til diskusjon: fornuftig/nødvendig?
```

Comment:
Synes det er bra at dere tok med hull i periode fra 07 til 09, for det blir jo mer virkelighetsnært, såvidt jeg har fått med meg.